WordPress/src/main/res/layout/domain_suggestions_fragment.xml

```diff
@@ -43,7 +43,7 @@
             android:background="@android:color/transparent"
             android:clickable="false"
             android:hint="@string/suggestions_search_hint"
-            android:imeOptions="actionSearch"
+            android:imeOptions="actionDone"
```

Comment:
This is a minor bug fix.
I changed to `actionDone`, since `actionSearch` will not dismiss the keyboard automatically on keypress.

Alternatively, I could also implement a `TextView.OnEditorActionListener` on this `EditText` to capture the event and dismiss the keyboard myself. What do you think?

Reply:
I prefer using actionDone, thanks for the clarification.